pages/styles/challenge.css

```diff
@@ -0,0 +1,40 @@
+/* TODO: Move <code> styles to global stylesheet */
```

Comment:
Can you implement this TODO and remove the comment by moving the `code` stylings to `common/styles/globalStyles.css`?